batch7crud-roll7/src/main/java/com/lftechnology/batch7crud/service/StudentServicesInterface.java

```diff
@@ -0,0 +1,24 @@
+package com.lftechnology.batch7crud.service;
+
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.exception.ValidatorException;
+import com.lftechnology.batch7crud.model.Student;
+
+import java.util.List;
+
+/**
+ * Created by leapfrog on 1/29/16.
+ */
+public interface StudentServicesInterface {
+  void addNew(Student student) throws DataException, ValidatorException;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: !unknownSymbol!, !unknownSymbol! [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)